charts/traefik/values.insecure.yaml.gotmpl

```diff
@@ -32,16 +33,18 @@ extraObjects:
   kind: Ingress
   metadata:
     name: traefik-dashboard
+    namespace: traefik
     annotations:
-      traefik.ingress.kubernetes.io/router.entrypoints: web,websecure  # allow http(s) for local deployment
-      traefik.ingress.kubernetes.io/router.middlewares: default-traefik-basic-auth@kubernetescrd  # namespace + middleware name
+      traefik.ingress.kubernetes.io/router.entrypoints: websecure  # allow http(s) for local deployment
```

Comment:
Why disallow `http` for local deployment? What benefits does it bring?

Reply:
I didnt see any use for it, but we can keep it as well I dont mind either way :--)